lib/githooks/pre-commit

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash -e
```

Reply:
What's the significance of this? vs set -e?